lib/core/network/api_service.dart

```diff
@@ -157,4 +157,32 @@ class APIService {
       return true;
     }
   }
+
+  Future<List<dynamic>> fetchHistory(
+      String childId, String type, int pageNr) async {
```

Comment:
can we follow an approach of other functions here, when body params goes here as map?
like:
Future<List<dynamic>> fetchHistory(
      String childId, Map<String, dynamic> body)